odpf/meta/common/Resource.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package odpf.meta.common;
+
+option java_package = "io.odpf.proton.meta.common";
+option java_outer_classname = "Resource";
+option go_package = "github.com/odpf/proton/meta/common";
+
+message Resource {
+  string urn = 1;
+  string name = 2;
+  string source = 3;
```

Comment:
can you give an example of what this source will contain?

Reply:
BigQuery, Kafka etc.